websocket.go

```diff
@@ -584,6 +584,7 @@ func (c *Client) SendProtobufParallel(nodes []*network.ServerIdentity, msg inter
 					finish.Do(func() { close(nodesChan) })
 					siChan <- node
 					replyChan <- reply
+					return
```

Comment:
just to check my understanding: the go-routine will return anyway if we succeed because `nodesChan` is closed so we will return on line 576, is that right?

Reply:
that's what I thought, but for some reason it doesn't :( At least as seen by my tests...

Reply:
I'll try it again, you're right, the `return` looks superfluous!